### ISSUE TITLE
Determine formats for different requests

### ISSUE THREAD

**Issue by yx2512** [devops-wishlist-buddies/wishlists#17]

**Comment by ntlz** [comment]
Great work Yu! I think we should do wishlist_id instead of wishlist_name, since the name is a user controlled string, and could change, so it would be more robust to operate with ids, like you did with users.
I also don't understand the function of select/deselect and update, could you elaborate?
Once this is finalized each URL and body should be copied into the descriptions of corresponding user stories.
Thanks!

**Comment by yx2512** [comment]
@ntlz That's definitely a good point. I will change those to ids instead.

In terms of select and deselect, it probably makes sense if we allow users to select all items inside a wishlist and see how much do they cost in total? Please let me know your thoughts on this.

My original thought for the "update" was that we could let users decide the number of items they would like to put inside a wishlist. However, since this is only a "wishlist", it doesn't make too much sense. I will remove that.

**Comment by ntlz** [comment]
@yx2512 awesome, thanks!
I'm not sure about the total cost of items, we can leave it for now.
It looks like you would prefer everything to be inside URL, without the body? If we could have a body of the request, then add item to wishlist could turn into POST /wishlists/add with body `{user_id: ...,  wishlist_id: items: [] }`, allowing the functionality of adding multiple items that you brought up here.

**Comment by yx2512** [comment]
@ntlz I totally agree with you. I was thinking about put URL arguments into request body (eg.content-type: application/json) as well. I wrote them in this way just to make sure that we can talk about this during our next meeting. I'm not sure whether the other teammates have any experience with web development or not and I don't want them to be confused at this time. I will talk about this during our next meeting and switch to request body implementation.

**Comment by ntlz** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/wishlists-614df33bb025e9001866fedf/issues/devops-wishlist-buddies/wishlists/17?planning-poker) with ZenHub @Fivetoes @jiuliwei @XinyuC97 @yx2512 @ZX-WANG
